docs/guides/using-the-network-shim.md

```diff
@@ -0,0 +1,288 @@
+# Using the Network Shim
+
+#### Table of content
+
+-   **[About](#about)**
+-   **[Quick setup](#quick-setup)**
+-   **[General workflow](#general-workflow)**
+-   **[Configuration](#configuration)**
+    -   **[Cypress](#cypress)**
+        -   **[Environment variables](#environment-variables)**
+        -   **[Plugins file](#plugins-file)**
+        -   **[Support file](#support-file)**
+    -   **[Package scripts and dependencies](#package-scripts-and-dependencies)**
+    -   **[GitHub workflow](#github-workflow)**
+
+## About
+
+The Network Shim makes it possible to record and play back network traffic during a cypress tests run. This means no live network requests need to be made when running e2e tests on CI, which is generally considered an anti-pattern as it can lead to flaky tests.
+
+It consists of a cypress plugin as well as a setup utility to be placed in cypress' support file. These have to be used in conjunction. Under the hood it uses the cypress `cy.intercept()` API to intercept network traffic during a test run. It has three different _modes_:
+
+-   `capture`: when operating in this mode, the Network Shim will intercept all outgoing requests to the specified hosts and allow each request to reach the destination server. Each response is stored as a fixture (JSON file).
+-   `stub`: in this mode the Network Shim also intercepts all outgoing requests to the specified hosts. However, it will prevent the request from reaching the destination server and instead it will reply with a response from a previously recorded fixture.
+-   `live`: the Network Shim does not intercept any requests when this mode is enabled.
```

Comment:
I was thinking this might clarify it a little:

> the Network Shim does not interfere with any network traffic when this mode is enabled. All network traffic is live.